[Bug] docker compose up failed on mac with orbstack

**Describe the Bug**
docker compose up failed on mac with orbstack

**To Reproduce**
Steps to reproduce the issue:
1. Configure docker with orbstack
2. copy .env
3. docker compose up

**Expected Behavior**
Run

**Screenshots**
If applicable, add screenshots or copies of the command line output to help explain the issue.

**Environment (please complete the following information):**
- OS: [e.g. macOS, Linux, Windows] macOS apple silicon
- Firecrawl Version: [e.g. 1.2.3] commit 2b7b7400f69a5a53dc248d24544b7092e1c85b41
- Node.js Version: [e.g. 14.x]

**Logs**

```
docker compose up                                                                                                                                                                         02:54:53 am
WARN[0000] The "PROXY_SERVER" variable is not set. Defaulting to a blank string.
WARN[0000] The "PROXY_USERNAME" variable is not set. Defaulting to a blank string.
WARN[0000] The "PROXY_PASSWORD" variable is not set. Defaulting to a blank string.
WARN[0000] The "BLOCK_MEDIA" variable is not set. Defaulting to a blank string.
WARN[0000] The "LOGTAIL_KEY" variable is not set. Defaulting to a blank string.
WARN[0000] The "POSTHOG_API_KEY" variable is not set. Defaulting to a blank string.
WARN[0000] The "POSTHOG_HOST" variable is not set. Defaulting to a blank string.
WARN[0000] The "SUPABASE_SERVICE_TOKEN" variable is not set. Defaulting to a blank string.
WARN[0000] The "SCRAPING_BEE_API_KEY" variable is not set. Defaulting to a blank string.
WARN[0000] The "NUM_WORKERS_PER_QUEUE" variable is not set. Defaulting to a blank string.
WARN[0000] The "LOGGING_LEVEL" variable is not set. Defaulting to a blank string.
WARN[0000] The "BULL_AUTH_KEY" variable is not set. Defaulting to a blank string.
WARN[0000] The "OPENAI_API_KEY" variable is not set. Defaulting to a blank string.
WARN[0000] The "SLACK_WEBHOOK_URL" variable is not set. Defaulting to a blank string.
WARN[0000] The "LLAMAPARSE_API_KEY" variable is not set. Defaulting to a blank string.
WARN[0000] The "SUPABASE_ANON_TOKEN" variable is not set. Defaulting to a blank string.
WARN[0000] The "USE_DB_AUTHENTICATION" variable is not set. Defaulting to a blank string.
WARN[0000] The "OPENAI_BASE_URL" variable is not set. Defaulting to a blank string.
WARN[0000] The "TEST_API_KEY" variable is not set. Defaulting to a blank string.
WARN[0000] The "SUPABASE_URL" variable is not set. Defaulting to a blank string.
WARN[0000] The "SELF_HOSTED_WEBHOOK_URL" variable is not set. Defaulting to a blank string.
WARN[0000] The "TEST_API_KEY" variable is not set. Defaulting to a blank string.
WARN[0000] The "SUPABASE_ANON_TOKEN" variable is not set. Defaulting to a blank string.
WARN[0000] The "NUM_WORKERS_PER_QUEUE" variable is not set. Defaulting to a blank string.
WARN[0000] The "OPENAI_API_KEY" variable is not set. Defaulting to a blank string.
WARN[0000] The "SCRAPING_BEE_API_KEY" variable is not set. Defaulting to a blank string.
WARN[0000] The "SLACK_WEBHOOK_URL" variable is not set. Defaulting to a blank string.
WARN[0000] The "LLAMAPARSE_API_KEY" variable is not set. Defaulting to a blank string.
WARN[0000] The "BULL_AUTH_KEY" variable is not set. Defaulting to a blank string.
WARN[0000] The "POSTHOG_HOST" variable is not set. Defaulting to a blank string.
WARN[0000] The "USE_DB_AUTHENTICATION" variable is not set. Defaulting to a blank string.
WARN[0000] The "LOGTAIL_KEY" variable is not set. Defaulting to a blank string.
WARN[0000] The "POSTHOG_API_KEY" variable is not set. Defaulting to a blank string.
WARN[0000] The "SUPABASE_URL" variable is not set. Defaulting to a blank string.
WARN[0000] The "SUPABASE_SERVICE_TOKEN" variable is not set. Defaulting to a blank string.
WARN[0000] The "OPENAI_BASE_URL" variable is not set. Defaulting to a blank string.
WARN[0000] The "SELF_HOSTED_WEBHOOK_URL" variable is not set. Defaulting to a blank string.
WARN[0000] The "LOGGING_LEVEL" variable is not set. Defaulting to a blank string.
[+] Running 9/9
 ✔ redis Pulled                                                                                                                                                                                                          7.5s
   ✔ 52f827f72350 Pull complete                                                                                                                                                                                          1.2s
   ✔ b3eda723fc9b Pull complete                                                                                                                                                                                          1.3s
   ✔ 572a44f2e990 Pull complete                                                                                                                                                                                          1.3s
   ✔ d7b5fcaf70e1 Pull complete                                                                                                                                                                                          1.8s
   ✔ 55763761dd3d Pull complete                                                                                                                                                                                          3.0s
   ✔ a5cd136d59c0 Pull complete                                                                                                                                                                                          3.0s
   ✔ 4f4fb700ef54 Pull complete                                                                                                                                                                                          3.0s
   ✔ 4ed994dafae2 Pull complete                                                                                                                                                                                          3.2s
[+] Building 289.6s (36/39)                                                                                                                                                                                      docker:orbstack
 => [playwright-service internal] load build definition from Dockerfile                                                                                                                                                     0.0s
 => => transferring dockerfile: 1.20kB                                                                                                                                                                                      0.0s
 => [playwright-service internal] load metadata for docker.io/library/python:3.11-slim                                                                                                                                      6.4s
 => [playwright-service internal] load .dockerignore                                                                                                                                                                        0.0s
 => => transferring context: 2B                                                                                                                                                                                             0.0s
 => [playwright-service internal] load build context                                                                                                                                                                        0.1s
 => => transferring context: 10.07kB                                                                                                                                                                                        0.0s
 => [playwright-service 1/6] FROM docker.io/library/python:3.11-slim@sha256:42420f737ba91d509fc60d5ed65ed0492678a90c561e1fa08786ae8ba8b52eda                                                                                5.3s
 => => resolve docker.io/library/python:3.11-slim@sha256:42420f737ba91d509fc60d5ed65ed0492678a90c561e1fa08786ae8ba8b52eda                                                                                                   0.0s
 => => sha256:1af95936e70e21da0df6a88fba09db5030f4fb39558363af95eafc1f03e26f15 16.13MB / 16.13MB                                                                                                                            2.9s
 => => sha256:42420f737ba91d509fc60d5ed65ed0492678a90c561e1fa08786ae8ba8b52eda 9.13kB / 9.13kB                                                                                                                              0.0s
 => => sha256:d93c300c71d98d0475da388d757c0e597c48e7205a735dd139c60fc32898fc64 1.75kB / 1.75kB                                                                                                                              0.0s
 => => sha256:552269f3e7d4390a1c089de001de97eb3e41cf533743d708237117659dbd0ff8 5.31kB / 5.31kB                                                                                                                              0.0s
 => => sha256:4d2547c084994a809c138e688fbe4ee14eedbc6e2defc5b1c680edd16e291473 28.04MB / 28.04MB                                                                                                                            4.2s
 => => sha256:93aa4e5d5dd64d3d056de9a6b0264c88e1fea27495bb2ec64859a43cdbf4a556 3.33MB / 3.33MB                                                                                                                              1.4s
 => => sha256:030ad528ba7029a4d6267c937fe1290c9b5dce07ea6cf2588f54e9814ccce7c4 249B / 249B                                                                                                                                  2.0s
 => => extracting sha256:4d2547c084994a809c138e688fbe4ee14eedbc6e2defc5b1c680edd16e291473                                                                                                                                   0.6s
 => => extracting sha256:93aa4e5d5dd64d3d056de9a6b0264c88e1fea27495bb2ec64859a43cdbf4a556                                                                                                                                   0.0s
 => => extracting sha256:1af95936e70e21da0df6a88fba09db5030f4fb39558363af95eafc1f03e26f15                                                                                                                                   0.3s
 => => extracting sha256:030ad528ba7029a4d6267c937fe1290c9b5dce07ea6cf2588f54e9814ccce7c4                                                                                                                                   0.0s
 => [playwright-service 2/6] RUN apt-get update && apt-get install -y --no-install-recommends     gcc     libstdc++6                                                                                                        7.7s
 => [playwright-service 3/6] WORKDIR /app                                                                                                                                                                                   0.0s
 => [playwright-service 4/6] COPY requirements.txt ./                                                                                                                                                                       0.0s
 => [playwright-service 5/6] RUN pip install --no-cache-dir --upgrade -r requirements.txt &&     pip uninstall -y py &&     playwright install chromium && playwright install-deps chromium &&     ln -s /usr/local/bin/  143.2s
 => [playwright-service 6/6] COPY . ./                                                                                                                                                                                      0.1s
 => [playwright-service] exporting to image                                                                                                                                                                                 1.4s
 => => exporting layers                                                                                                                                                                                                     1.3s
 => => writing image sha256:651a3b7dc33e274be5339f77e86ab422161d4d71a54d9f792a3ce39beec08f72                                                                                                                                0.0s
 => => naming to docker.io/library/firecrawl-playwright-service                                                                                                                                                             0.0s
 => [playwright-service] resolving provenance for metadata file                                                                                                                                                             0.0s
 => [api internal] load build definition from Dockerfile                                                                                                                                                                    0.0s
 => => transferring dockerfile: 2.15kB                                                                                                                                                                                      0.0s
 => [api internal] load metadata for docker.io/library/node:20-slim                                                                                                                                                         3.1s
 => [api internal] load metadata for docker.io/library/rust:1-bullseye                                                                                                                                                      3.5s
 => [api internal] load metadata for docker.io/library/golang:1.19                                                                                                                                                          3.6s
 => [api internal] load .dockerignore                                                                                                                                                                                       0.0s
 => => transferring context: 110B                                                                                                                                                                                           0.0s
 => [api base 1/5] FROM docker.io/library/node:20-slim@sha256:4ada13d4258db3809cbff56d605f80af8383bf1f823168d0518d8dce799e7cf0                                                                                             62.9s
 => => resolve docker.io/library/node:20-slim@sha256:4ada13d4258db3809cbff56d605f80af8383bf1f823168d0518d8dce799e7cf0                                                                                                       0.0s
 => => sha256:4ada13d4258db3809cbff56d605f80af8383bf1f823168d0518d8dce799e7cf0 6.49kB / 6.49kB                                                                                                                              0.0s
 => => sha256:9e23f4e30758ea1f59fcf8ef8bc389b4b2024bed1e7d7d6f10b9b0cc4700cdee 1.93kB / 1.93kB                                                                                                                              0.0s
 => => sha256:064fe8d38fc28e4d35af03da126e109e36a3ee798a97df0a327f932ad00aa124 6.56kB / 6.56kB                                                                                                                              0.0s
 => => sha256:cf0e2edd2ee8d65bcbdb085a6ca0d56366857b568ddc43b647f4306ec7e587a6 3.32kB / 3.32kB                                                                                                                             42.7s
 => => extracting sha256:cf0e2edd2ee8d65bcbdb085a6ca0d56366857b568ddc43b647f4306ec7e587a6                                                                                                                                   0.0s
 => => sha256:4d155826916281acbfceefee226dfa2f075e5ed3549251159f4ab3e204e223b1 40.73MB / 40.73MB                                                                                                                           60.5s
 => => extracting sha256:4d155826916281acbfceefee226dfa2f075e5ed3549251159f4ab3e204e223b1                                                                                                                                   0.6s
 => => sha256:a257348e90991e8c1ea7f8014995efde63b26ac1ceae0e254b10fcbb9c570cb6 1.71MB / 1.71MB                                                                                                                             62.4s
 => => extracting sha256:a257348e90991e8c1ea7f8014995efde63b26ac1ceae0e254b10fcbb9c570cb6                                                                                                                                   0.0s
 => => sha256:50095d43980034cdf5680b5ce32cb5193661b22f6d02cc7e59511fdb90a1f2f4 444B / 444B                                                                                                                                 62.7s
 => => extracting sha256:50095d43980034cdf5680b5ce32cb5193661b22f6d02cc7e59511fdb90a1f2f4                                                                                                                                   0.0s
 => [api internal] load build context                                                                                                                                                                                       0.1s
 => => transferring context: 1.93MB                                                                                                                                                                                         0.0s
 => [api go-base 1/3] FROM docker.io/library/golang:1.19@sha256:3025bf670b8363ec9f1b4c4f27348e6d9b7fec607c47e401e40df816853e743a                                                                                           37.4s
 => => resolve docker.io/library/golang:1.19@sha256:3025bf670b8363ec9f1b4c4f27348e6d9b7fec607c47e401e40df816853e743a                                                                                                        0.0s
 => => sha256:3025bf670b8363ec9f1b4c4f27348e6d9b7fec607c47e401e40df816853e743a 2.36kB / 2.36kB                                                                                                                              0.0s
 => => sha256:df70a71c3fbe29bde497ca937b8e0cb93d83e359f641209a7848e1d8cfffd8a0 6.88kB / 6.88kB                                                                                                                              0.0s
 => => sha256:993525952a483617183655b735562fd3eee4b3a85a472e3dc3c79a258adca100 23.57MB / 23.57MB                                                                                                                           13.6s
 => => sha256:12916f73f9de8e251c4eb4f2d079fa6cd63205b7ba8a7bd88e1f9446105e9ef1 63.99MB / 63.99MB                                                                                                                            7.9s
 => => sha256:b43f8231bf8e9cfe1e388ff2ed95e11e1f63a2275eb44fa7a37d95169aa36c78 1.58kB / 1.58kB                                                                                                                              0.0s
 => => sha256:fa34261389f3dee2ecffe9bfe38ae7fd05a1908e98a7486354905aee0f648a6e 49.59MB / 49.59MB                                                                                                                           20.9s
 => => sha256:0b84cd7f173d9c2c1287768f308d87cd738f9b833164dafc86f19453b6204a6e 86.30MB / 86.30MB                                                                                                                           23.0s
 => => sha256:d20a116de072ace539a45fe44925ff9e0a1127b68fd9bd6a6583830b057be185 115.33MB / 115.33MB                                                                                                                         34.9s
 => => extracting sha256:fa34261389f3dee2ecffe9bfe38ae7fd05a1908e98a7486354905aee0f648a6e                                                                                                                                   0.8s
 => => sha256:3b30480db207227e6571e4e4312eddbe2990ce9169981907b30b27eff1dc79f5 154B / 154B                                                                                                                                 22.0s
 => => extracting sha256:993525952a483617183655b735562fd3eee4b3a85a472e3dc3c79a258adca100                                                                                                                                   0.2s
 => => extracting sha256:12916f73f9de8e251c4eb4f2d079fa6cd63205b7ba8a7bd88e1f9446105e9ef1                                                                                                                                   1.0s
 => => extracting sha256:0b84cd7f173d9c2c1287768f308d87cd738f9b833164dafc86f19453b6204a6e                                                                                                                                   1.1s
 => => extracting sha256:d20a116de072ace539a45fe44925ff9e0a1127b68fd9bd6a6583830b057be185                                                                                                                                   2.1s
 => => extracting sha256:3b30480db207227e6571e4e4312eddbe2990ce9169981907b30b27eff1dc79f5                                                                                                                                   0.0s
 => [api rust-base 1/3] FROM docker.io/library/rust:1-bullseye@sha256:1e3f7a9fd1f278cc4be02a830745f40fe4b22f0114b2464a452c50273cc1020d                                                                                    103.8s
 => => resolve docker.io/library/rust:1-bullseye@sha256:1e3f7a9fd1f278cc4be02a830745f40fe4b22f0114b2464a452c50273cc1020d                                                                                                    0.0s
 => => sha256:1e3f7a9fd1f278cc4be02a830745f40fe4b22f0114b2464a452c50273cc1020d 5.20kB / 5.20kB                                                                                                                              0.0s
 => => sha256:1b3ed2041a9c483ddc433dc9194cce9aa4e9bfd5b20a35184027ada6519464cd 1.94kB / 1.94kB                                                                                                                              0.0s
 => => sha256:72bc64f5fae755ee6198b20cc007940776f9120decf2cfdd568f7f2fce8b9afa 4.09kB / 4.09kB                                                                                                                              0.0s
 => => sha256:0038ef039a89ede34c57806e684dc9d9be7dcd22d3c08b90deb36bb22a2c7122 52.25MB / 52.25MB                                                                                                                           33.2s
 => => sha256:82c7afb1aa0f9672a06c4c7eaa6b7c7e111a91a8d45272dce1e361ac0b0ed79a 15.54MB / 15.54MB                                                                                                                           27.6s
 => => sha256:bf8e2f45c7ddf8cc116eeb2ac1ef8be70e3639a883c6d9e5eaf1f2dd702dbf92 54.85MB / 54.85MB                                                                                                                           41.8s
 => => extracting sha256:0038ef039a89ede34c57806e684dc9d9be7dcd22d3c08b90deb36bb22a2c7122                                                                                                                                   0.9s
 => => sha256:2cae5cdedf5465f9b64d9afa67592e36727e50a811baf4a0d35508744cd435c3 189.98MB / 189.98MB                                                                                                                         96.4s
 => => extracting sha256:82c7afb1aa0f9672a06c4c7eaa6b7c7e111a91a8d45272dce1e361ac0b0ed79a                                                                                                                                   0.2s
 => => sha256:eb22a1c999d9118c6e8867313c8c862e8e404aedf8997625f9c60334e8304c11 258.31MB / 258.31MB                                                                                                                         63.3s
 => => extracting sha256:bf8e2f45c7ddf8cc116eeb2ac1ef8be70e3639a883c6d9e5eaf1f2dd702dbf92                                                                                                                                   1.0s
 => => extracting sha256:2cae5cdedf5465f9b64d9afa67592e36727e50a811baf4a0d35508744cd435c3                                                                                                                                   3.0s
 => => extracting sha256:eb22a1c999d9118c6e8867313c8c862e8e404aedf8997625f9c60334e8304c11                                                                                                                                   3.8s
 => [api go-base 2/3] COPY sharedLibs/go-html-to-md /app/sharedLibs/go-html-to-md                                                                                                                                           1.0s
 => [api go-base 3/3] RUN cd /app/sharedLibs/go-html-to-md &&     go mod tidy &&     go build -o html-to-markdown.so -buildmode=c-shared html-to-markdown.go &&     chmod +x html-to-markdown.so                           11.9s
 => [api base 2/5] RUN npm i -g corepack@latest                                                                                                                                                                             1.4s
 => [api base 3/5] RUN corepack enable                                                                                                                                                                                      0.1s
 => [api base 4/5] COPY . /app                                                                                                                                                                                              0.0s
 => [api base 5/5] WORKDIR /app                                                                                                                                                                                             0.0s
 => [api prod-deps 1/1] RUN --mount=type=cache,id=pnpm,target=/pnpm/store pnpm install --prod --frozen-lockfile                                                                                                            32.3s
 => [api build 1/4] RUN --mount=type=cache,id=pnpm,target=/pnpm/store pnpm install --frozen-lockfile                                                                                                                       32.3s
 => [api build 2/4] RUN apt-get clean && apt-get update -qq && apt-get install -y ca-certificates && update-ca-certificates                                                                                                 4.8s
 => [api stage-5 1/5] COPY --from=prod-deps /app/node_modules /app/node_modules                                                                                                                                             2.5s
 => [api build 3/4] RUN pnpm install                                                                                                                                                                                        2.1s
 => [api build 4/4] RUN --mount=type=secret,id=SENTRY_AUTH_TOKEN     bash -c 'export SENTRY_AUTH_TOKEN="$(cat /run/secrets/SENTRY_AUTH_TOKEN)"; if [ -z $SENTRY_AUTH_TOKEN ]; then pnpm run build:nosentry; else pnpm run   7.1s
 => [api rust-base 2/3] COPY sharedLibs/html-transformer /app/sharedLibs/html-transformer                                                                                                                                   0.2s
 => ERROR [api rust-base 3/3] RUN cd /app/sharedLibs/html-transformer &&     cargo build --release &&     chmod +x target/release/libhtml_transformer.so                                                                   17.0s
 => CANCELED [api stage-5 2/5] COPY --from=build /app /app                                                                                                                                                                  6.1s
------
 > [api rust-base 3/3] RUN cd /app/sharedLibs/html-transformer &&     cargo build --release &&     chmod +x target/release/libhtml_transformer.so:
0.162     Updating crates.io index
3.379  Downloading crates ...
3.702   Downloaded fxhash v0.2.1
3.717   Downloaded convert_case v0.4.0
3.719   Downloaded getrandom v0.2.15
3.721   Downloaded autocfg v1.4.0
3.727   Downloaded equivalent v1.0.1
3.736   Downloaded zerovec-derive v0.10.3
3.740   Downloaded writeable v0.5.5
3.746   Downloaded nodrop v0.1.14
3.747   Downloaded matches v0.1.10
3.747   Downloaded icu_provider_macros v1.5.0
3.748   Downloaded stable_deref_trait v1.2.0
3.749   Downloaded mac v0.1.1
3.752   Downloaded phf_codegen v0.10.0
3.753   Downloaded cfg-if v1.0.0
3.755   Downloaded phf_generator v0.8.0
3.758   Downloaded phf_macros v0.10.0
3.761   Downloaded precomputed-hash v0.1.1
3.761   Downloaded phf v0.8.0
3.765   Downloaded phf_shared v0.10.0
3.766   Downloaded phf_codegen v0.8.0
3.771   Downloaded new_debug_unreachable v1.0.6
3.772   Downloaded write16 v1.0.0
3.772   Downloaded yoke-derive v0.7.5
3.776   Downloaded utf8_iter v1.0.4
3.777   Downloaded string_cache_codegen v0.5.2
3.777   Downloaded phf_shared v0.8.0
3.778   Downloaded phf_generator v0.10.0
3.779   Downloaded thin-slice v0.1.1
3.779   Downloaded form_urlencoded v1.2.1
3.780   Downloaded dtoa v1.0.9
3.781   Downloaded cssparser-macros v0.6.1
3.781   Downloaded displaydoc v0.2.5
3.783   Downloaded phf_macros v0.8.0
3.785   Downloaded rand_chacha v0.2.2
3.789   Downloaded byteorder v1.5.0
3.791   Downloaded foldhash v0.1.4
3.791   Downloaded dtoa-short v0.3.5
3.793   Downloaded futf v0.1.5
3.797   Downloaded rand_chacha v0.3.1
3.802   Downloaded itoa v1.0.14
3.810   Downloaded percent-encoding v2.3.1
3.814   Downloaded rand_core v0.5.1
3.827   Downloaded servo_arc v0.1.1
3.828   Downloaded mime v0.3.17
3.830   Downloaded phf v0.10.1
3.831   Downloaded zerofrom v0.1.5
3.835   Downloaded tinystr v0.7.6
3.839   Downloaded siphasher v0.3.11
3.840   Downloaded scopeguard v1.2.0
3.841   Downloaded rustc_version v0.4.1
3.842   Downloaded idna_adapter v1.2.0
3.845   Downloaded proc-macro-hack v0.5.20+deprecated
3.846   Downloaded bitflags v1.3.2
3.849   Downloaded rand_pcg v0.2.1
3.851   Downloaded itoa v0.4.8
3.853   Downloaded synstructure v0.13.1
3.854   Downloaded servo_arc v0.2.0
3.856   Downloaded string_cache v0.8.7
3.862   Downloaded thiserror-impl v2.0.11
3.865   Downloaded getrandom v0.1.16
3.869   Downloaded lock_api v0.4.12
3.870   Downloaded icu_locid_transform v1.5.0
3.876   Downloaded once_cell v1.20.2
3.878   Downloaded semver v1.0.25
3.880   Downloaded ppv-lite86 v0.2.20
3.881   Downloaded markup5ever v0.11.0
3.883   Downloaded litemap v0.7.4
3.887   Downloaded rand_core v0.6.4
3.889   Downloaded kuchikiki v0.8.2
3.897   Downloaded smallvec v1.13.2
3.904   Downloaded parking_lot_core v0.9.10
3.909   Downloaded tendril v0.4.3
3.913   Downloaded utf16_iter v1.0.5
3.917   Downloaded derive_more v0.99.18
3.920   Downloaded quote v1.0.38
3.921   Downloaded cssparser v0.29.6
3.923   Downloaded icu_normalizer v1.5.0
3.927   Downloaded zerofrom-derive v0.1.5
3.927   Downloaded utf-8 v0.7.6
3.929   Downloaded thiserror v2.0.11
3.932   Downloaded yoke v0.7.5
3.933   Downloaded log v0.4.25
3.935   Downloaded icu_provider v1.5.0
3.937   Downloaded icu_locid v1.5.0
3.940   Downloaded bitflags v2.8.0
3.943   Downloaded zerocopy-derive v0.7.35
3.947   Downloaded icu_locid_transform_data v1.5.0
3.951   Downloaded unicode-ident v1.0.15
3.953   Downloaded selectors v0.22.0
3.954   Downloaded zerovec v0.10.4
3.959   Downloaded icu_normalizer_data v1.5.0
3.960   Downloaded url v2.5.4
3.963   Downloaded zerocopy v0.7.35
3.972   Downloaded selectors v0.24.0
3.973   Downloaded indexmap v1.9.3
3.975   Downloaded serde_derive v1.0.217
3.977   Downloaded html5ever v0.26.0
3.979   Downloaded parking_lot v0.12.3
3.981   Downloaded cssparser v0.27.2
3.983   Downloaded icu_properties v1.5.1
3.985   Downloaded proc-macro2 v1.0.93
3.987   Downloaded ryu v1.0.18
3.989   Downloaded allocator-api2 v0.2.21
3.995   Downloaded icu_collections v1.5.0
4.007   Downloaded rand v0.8.5
4.011   Downloaded serde v1.0.217
4.014   Downloaded memchr v2.7.4
4.019   Downloaded hashbrown v0.12.3
4.031   Downloaded rand v0.7.3
4.039   Downloaded idna v1.0.3
4.043   Downloaded hashbrown v0.15.2
4.048   Downloaded serde_json v1.0.137
4.053   Downloaded lol_html v2.2.0
4.080   Downloaded icu_properties_data v1.5.0
4.087   Downloaded syn v1.0.109
4.099   Downloaded syn v2.0.96
4.167   Downloaded libc v0.2.169
4.210   Downloaded encoding_rs v0.8.35
4.271    Compiling proc-macro2 v1.0.93
4.271    Compiling unicode-ident v1.0.15
4.271    Compiling libc v0.2.169
4.271    Compiling cfg-if v1.0.0
4.271    Compiling byteorder v1.5.0
4.271    Compiling siphasher v0.3.11
4.271    Compiling syn v1.0.109
4.272    Compiling getrandom v0.1.16
4.273    Compiling stable_deref_trait v1.2.0
4.274    Compiling smallvec v1.13.2
4.274    Compiling proc-macro-hack v0.5.20+deprecated
4.371    Compiling autocfg v1.4.0
4.383    Compiling writeable v0.5.5
4.492    Compiling phf_shared v0.8.0
4.510    Compiling phf_shared v0.10.0
4.570    Compiling litemap v0.7.4
4.666    Compiling serde v1.0.217
4.666    Compiling precomputed-hash v0.1.1
4.679    Compiling parking_lot_core v0.9.10
4.713    Compiling dtoa v1.0.9
4.758    Compiling scopeguard v1.2.0
4.773    Compiling lock_api v0.4.12
4.809    Compiling nodrop v0.1.14
4.858    Compiling dtoa-short v0.3.5
4.859    Compiling matches v0.1.10
4.902    Compiling log v0.4.25
4.913    Compiling icu_locid_transform_data v1.5.0
4.939    Compiling new_debug_unreachable v1.0.6
4.960    Compiling convert_case v0.4.0
4.969    Compiling icu_properties_data v1.5.0
4.976    Compiling mac v0.1.1
4.980    Compiling utf16_iter v1.0.5
5.025    Compiling futf v0.1.5
5.066    Compiling write16 v1.0.0
5.097    Compiling quote v1.0.38
5.113    Compiling once_cell v1.20.2
5.114    Compiling utf-8 v0.7.6
5.123    Compiling itoa v1.0.14
5.147    Compiling utf8_iter v1.0.4
5.192    Compiling fxhash v0.2.1
5.210    Compiling bitflags v1.3.2
5.244    Compiling icu_normalizer_data v1.5.0
5.283    Compiling tendril v0.4.3
5.293    Compiling indexmap v1.9.3
5.294    Compiling itoa v0.4.8
5.299    Compiling thiserror v2.0.11
5.305    Compiling syn v2.0.96
5.386    Compiling getrandom v0.2.15
5.414    Compiling servo_arc v0.1.1
5.433    Compiling rand_core v0.5.1
5.449    Compiling servo_arc v0.2.0
5.462    Compiling memchr v2.7.4
5.498    Compiling rand_core v0.6.4
5.504    Compiling serde_json v1.0.137
5.531    Compiling foldhash v0.1.4
5.560    Compiling rand_pcg v0.2.1
5.635    Compiling hashbrown v0.12.3
5.674    Compiling allocator-api2 v0.2.21
5.699    Compiling equivalent v1.0.1
5.701    Compiling percent-encoding v2.3.1
5.704    Compiling thin-slice v0.1.1
5.709    Compiling encoding_rs v0.8.35
5.738    Compiling mime v0.3.17
5.749    Compiling ryu v1.0.18
5.771    Compiling parking_lot v0.12.3
6.011    Compiling form_urlencoded v1.2.1
6.059    Compiling bitflags v2.8.0
6.164    Compiling hashbrown v0.15.2
8.390    Compiling synstructure v0.13.1
8.556    Compiling cssparser v0.29.6
8.558    Compiling cssparser v0.27.2
8.558    Compiling html5ever v0.26.0
8.989    Compiling zerocopy-derive v0.7.35
9.050    Compiling zerofrom-derive v0.1.5
9.099    Compiling yoke-derive v0.7.5
9.156    Compiling zerovec-derive v0.10.3
9.158    Compiling displaydoc v0.2.5
9.159    Compiling icu_provider_macros v1.5.0
9.182    Compiling serde_derive v1.0.217
9.229    Compiling cssparser-macros v0.6.1
9.653    Compiling derive_more v0.99.18
9.944    Compiling thiserror-impl v2.0.11
10.23    Compiling zerocopy v0.7.35
10.68    Compiling zerofrom v0.1.5
10.73    Compiling yoke v0.7.5
10.76    Compiling ppv-lite86 v0.2.20
10.87    Compiling zerovec v0.10.4
10.95    Compiling rand_chacha v0.2.2
10.95    Compiling rand_chacha v0.3.1
11.04    Compiling rand v0.7.3
11.07    Compiling rand v0.8.5
11.56    Compiling phf_generator v0.10.0
11.56    Compiling phf_generator v0.8.0
11.60    Compiling string_cache_codegen v0.5.2
11.60    Compiling phf_codegen v0.10.0
11.61    Compiling phf_codegen v0.8.0
11.63    Compiling phf_macros v0.10.0
11.66    Compiling tinystr v0.7.6
11.66    Compiling icu_collections v1.5.0
11.67    Compiling phf_macros v0.8.0
11.67    Compiling selectors v0.24.0
11.67    Compiling selectors v0.22.0
11.84    Compiling icu_locid v1.5.0
11.95    Compiling markup5ever v0.11.0
12.37    Compiling icu_provider v1.5.0
12.40    Compiling phf v0.10.1
12.42    Compiling phf v0.8.0
12.64    Compiling icu_locid_transform v1.5.0
13.13    Compiling icu_properties v1.5.1
13.80    Compiling lol_html v2.2.0
14.33    Compiling string_cache v0.8.7
14.67    Compiling icu_normalizer v1.5.0
15.10    Compiling idna_adapter v1.2.0
15.17    Compiling idna v1.0.3
15.53    Compiling url v2.5.4
15.80    Compiling kuchikiki v0.8.2
16.67    Compiling html-transformer v0.1.0 (/app/sharedLibs/html-transformer)
16.73 error[E0308]: mismatched types
16.73    --> src/lib.rs:379:37
16.73     |
16.73 379 |     drop(unsafe { CString::from_raw(ptr) })
16.73     |                   ----------------- ^^^ expected `*mut u8`, found `*mut i8`
16.73     |                   |
16.73     |                   arguments to this function are incorrect
16.73     |
16.73     = note: expected raw pointer `*mut u8`
16.73                found raw pointer `*mut i8`
16.73 note: associated function defined here
16.73    --> /rustc/e71f9a9a98b0faf423844bf0ba7438f29dc27d58/library/alloc/src/ffi/c_str.rs:396:19
16.73
16.77 For more information about this error, try `rustc --explain E0308`.
16.77 error: could not compile `html-transformer` (lib) due to 1 previous error
------
failed to solve: process "/bin/sh -c cd /app/sharedLibs/html-transformer &&     cargo build --release &&     chmod +x target/release/libhtml_transformer.so" did not complete successfully: exit code: 101
~/Apps/firecrawl main ❯
```

**Additional Context**
Add any other context about the problem here, such as configuration specifics, network conditions, data volumes, etc.

--- a/apps/api/sharedLibs/html-transformer/src/lib.rs
+++ b/apps/api/sharedLibs/html-transformer/src/lib.rs
@@ -375,6 +375,6 @@ pub unsafe extern "C" fn get_inner_json(html: *const libc::c_char) -> *mut libc:
 /// # Safety
 /// ptr must be a non-freed string pointer returned by Rust code.
 #[no_mangle]
-pub unsafe extern "C" fn free_string(ptr: *mut i8) {
+pub unsafe extern "C" fn free_string(ptr: *mut libc::c_char) {
     drop(unsafe { CString::from_raw(ptr) })
 }

